scripts/autobahn-server.sh

```diff
@@ -23,9 +22,13 @@ function test_diff() {
     fi
 }
 
-cargo build --release --example autobahn-server
 cargo run --release --example autobahn-server & WSSERVER_PID=$!
-echo "Server PID: ${WSSERVER_PID}"
 sleep 3
-wstest -m fuzzingclient -s 'autobahn/fuzzingclient.json'
+
+docker run -d --rm \
+    -v "${PWD}/autobahn:/autobahn" \
+    --network host \
```

Comment:
As an alternative to `--network host`, we can/should use `172.17.0.1` to contact the Docker host machine from within the container. (though this may not be a portable solution - see https://stackoverflow.com/questions/22944631/how-to-get-the-ip-address-of-the-docker-host-from-inside-a-docker-container)

Reply:
Yes, the portability was one of my concerns. That would also mean that the config must be adjusted, so that the fuzzingclient is configuredf to connect to `172.17.0.1`. Not a big deal provided we use it only in conjunction with docker in this particular configuration.

Reply:
I see, okay, feel free to keep `--network host` and resolve this thread, it is not a big deal.

Note that `--network host` is reportedly even less portable, per https://docs.docker.com/network/host/:
> The host networking driver only works on Linux hosts, and is not supported on Docker Desktop for Mac, Docker Desktop for Windows, or Docker EE for Windows Server.

(though some my friends say that Docker host networking works for them as expected on Macs)

Reply:
Oh, alright. Yeah, seems to work on my M1 Mac as well...